channel/src/main/java/com/synopsys/integration/alert/channel/msteams2/MSTeamsChannelMessageSender.java

```diff
@@ -0,0 +1,93 @@
+ * under the License.
+ */
+package com.synopsys.integration.alert.channel.msteams2;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.synopsys.integration.alert.channel.util.RestChannelUtility;
+import com.synopsys.integration.alert.common.exception.AlertException;
+import com.synopsys.integration.alert.common.message.model.MessageResult;
+import com.synopsys.integration.alert.common.persistence.model.job.details.MSTeamsJobDetailsModel;
+import com.synopsys.integration.alert.descriptor.api.MsTeamsKey;
+import com.synopsys.integration.rest.request.Request;
+import com.synopys.integration.alert.channel.api.ChannelMessageSender;
+
+@Component
+public class MSTeamsChannelMessageSender implements ChannelMessageSender<MSTeamsJobDetailsModel, MSTeamsChannelMessageModel, MessageResult> {
+    private final RestChannelUtility restChannelUtility;
+    private final MsTeamsKey msTeamsKey;
+
+    @Autowired
+    public MSTeamsChannelMessageSender(RestChannelUtility restChannelUtility, MsTeamsKey msTeamsKey) {
+        this.restChannelUtility = restChannelUtility;
+        this.msTeamsKey = msTeamsKey;
+    }
+
+    @Override
+    public MessageResult sendMessages(MSTeamsJobDetailsModel msTeamsJobDetailsModel, List<MSTeamsChannelMessageModel> channelMessages) throws AlertException {
+        String webhook = msTeamsJobDetailsModel.getWebhook();
+
+        String messageTitle = "Received a message from Black Duck"; // TODO: How should we get the provider? Each message converted in a ChannelMessageConverter is as ProviderMessage.
+        Map<String, String> requestHeaders = new HashMap<>();
+        requestHeaders.put("Content-Type", "application/json");
+
+        Request request = createRequestsForMessage(webhook, messageTitle, channelMessages, requestHeaders);
+
+        restChannelUtility.sendSingleMessage(request, msTeamsKey.getUniversalKey());
+
+        return new MessageResult(String.format("Successfully sent %d MSTeams message(s)", channelMessages.size()));
+    }
+
+    private Request createRequestsForMessage(String webhook, String messageTitle, List<MSTeamsChannelMessageModel> messages, Map<String, String> requestHeaders) {
+        String jsonString = getJsonString(messageTitle, messages);
+        return restChannelUtility.createPostMessageRequest(webhook, requestHeaders, jsonString);
+    }
+
+    private String getJsonString(String title, List<MSTeamsChannelMessageModel> messageSections) {
+        JsonObject json = new JsonObject();
+        json.addProperty("@type", "MessageCard");
+        json.addProperty("@context", "https://schema.org/extensions");
+        json.addProperty("summary", "New Content from Alert");
+        json.addProperty("themeColor", "5A2A82");
```

Comment:
I think at least the color hex code should be in a static variable with a name for the color.

Reply:
How's this?
```
    private static final String MESSAGE_THEME_COLOR = "5A2A82"; // Synopsys Purple
    private static final String MESSAGE_SUMMARY = "New Content from Alert";
 ```

Reply:
I think that's good. Adding the comment as part of the name may be better as you'll see the color directy where it's used.